src/e2ee.rs

```diff
@@ -379,4 +386,106 @@ Sent with my Delta Chat Messenger: https://delta.chat";
         let data = b"blas";
         assert_eq!(has_decrypted_pgp_armor(data), false);
     }
+
+    #[async_std::test]
+    async fn test_encrypted_no_autocrypt() -> crate::error::Result<()> {
```

Comment:
returning result in these tests looks like it works pretty nicely.  i should probably try this myself, i'd completely forgotten about this.